README.md

```diff
@@ -35,3 +35,29 @@ COUNTRY_PACKAGE=openfisca_country_template gunicorn "openfisca_web_api_preview.a
 ```
 
 The `--workers k` (with `k >= 3`) option is necessary to avoid [this issue](http://stackoverflow.com/questions/11150343/slow-requests-on-local-flask-server). Without it, AJAX requests from Chrome sometimes take more than 20s to process.
+
+### Tracker
+
+The OpenFisca Web API comes with an [optional tracker](https://github.com/openfisca/tracker) which allows you to measure the usage of the API.
+
+#### Tracker installation
+
+The tracker is not installed by default. To install it, run:
+
+```sh
+pip install openfisca_core[tracker]  # Or `pip install --editable ".[tracker]"` for an editable installation
+```
+
+
+#### Tracker configuration
+
+The tracker is activated when these two environment variables are set:
```

Comment:
> I wonder if it would not be simpler to just combine these two environment variable into one, for instance `TRACKER=https://stats.data.gouv.fr/piwik.php?idsite=2` 🤔